steward/src/commands/schedule_corks/cellar_v1/lift_shutdown.rs

```diff
@@ -1,7 +1,7 @@
 use crate::{application::APP, cellars, cork, prelude::*};
 use abscissa_core::{clap::Parser, Command, Runnable};
 use ethers::abi::AbiEncode;
-use steward_abi::cellar::*;
+use steward_abi::cellar_v1::*;
```

Comment:
Tagging this in a schedule command even though it's not relevant to the line: should we include scheduled cork calls for the governance functions of Cellar V2? Without them, we can't execute one of those functions in the intervening time before the upgrade.

Reply:
Since such a call would require coordination on the part of the validator set to use anyway, perhaps we can just upgrade this 2.x line with it if the issue comes up, and then have them upgrade/do the command at the same time?

Reply:
We can prep this code after cutting this release.